### DISCUSSION TITLE
Spanish

### DISCUSSION THREAD

**Discussion by mjm69** [epodegrid/epd106-ADB#8] [General]
Could It be possible in Spanish ?

**Comment by epodegrid** [comment]
The official firmware doesn't support it and it probably never will. If you're willing to take the risk, rooting the device is an option. 4PDA has good information and tools for this. Just as a note, rooting the device can have serious side effects (much more than phones) because of the lackluster hardware and software, so I wouldn't recommend it.

**Comment by leonidofsparta** [comment]
> The official firmware doesn't support it and it probably never will. If you're willing to take the risk, rooting the device is an option. 4PDA has good information and tools for this. Just as a note, rooting the device can have serious side effects (much more than phones) because of the lackluster hardware and software, so I wouldn't recommend it.

Actually there are no side effects at all until you use Xposed modules, as Magisk eats no resources. Also, it violates no warranty, because you can reupload stock firmware if you need to send it to warranty. 
And, if you concern is privacy or related (I have good points to believing this, as you just ignored me when I told you about the progress I mad in modifications) there is nothing to worry about (also because, who the hell loads credit card data or other sensible data on an ereader and why?).
